### DISCUSSION TITLE
[projects beta] View type not saved between page refreshes

### DISCUSSION THREAD

**Discussion by bduffany** [community/community#6490] [Projects and Issues]
When I refresh the page, my view that I've configured to be in "Board" view seems to revert back to a normal table view.

It looks like the view type is only being stored as a URL param, which would explain why this is happening.

https://user-images.githubusercontent.com/2414826/137778107-c0881700-52c2-4ff8-8d0d-50027611ccb7.mp4

**Comment by e0d** [comment]
You need to explicitly save changes from the carrot drop-down menu.  Typically there's a "dirty" indicator when you've made an unsaved change:

![image](https://user-images.githubusercontent.com/1270274/137818585-ccc60ac0-b4d9-4200-86f0-6dcb3961b7d8.png)
